tests/unit/backend/wmg/fixtures/test_snapshot.py

```diff
@@ -254,16 +252,12 @@ def create_cubes(
         data_dir, coords, dim_values, expression_summary_vals_fn=expression_summary_vals_fn
     )
 
-    expression_summary_fmg_cube_dir = create_expression_summary_fmg_cube(
-        data_dir, coords, dim_values, expression_summary_fmg_vals_fn=expression_summary_fmg_vals_fn
```

Comment:
I decided to not generate fake data artifacts for anything related to marker genes.

Instead, tests will use a test fixture snapshot containing subsetted, real data. Now, we can test the marker gene cube generation as well as the associated marker gene calculation code.

Reply:
A side benefit is that any test that uses `create_temp_wmg_snapshot` is much faster, now. Backend unit test times are cut in half after this reversion.

Reply:
I had not tested `expression_summary_fmg` generation, as we are testing neither `cell_counts` or `expression_summary` generation. Those tests exists but they were stubbed (i.e. just a `pass`).

`expression_summary_fmg` is used in two ways for tests:
1. test marker gene cube creation.
2. test query results (`test_query.py`).